src/md/src/WorkWithReports/UID.md

```diff
@@ -0,0 +1,18 @@
+## A test item UniqueID
+
+### UniqueID
+
+A UniqueID is used for identifying  a test item's originality. ReportPortal generates an ID automatically for all test items by default. 
+The UID is generated based on:
+
+*	A project name; 
+* A launch name;
+* Names of all parents;
+* All parameters of the item;
+
+All this information is encoded into MD5. After that a UID becomes a part of the item. It gives an opportunity to define the item uniqueness with no possibility of doubt. 
+ReportPortal plans to use this functionality in a process of building widgets ( f.e.:`Most failed test cases`, `Flack test`), `Rety` and `Rerun`, `Merge` and etc.
+
+You can provide a custom ID for a test item using the annotation as well. 
```

Comment:
annotations work only for java clients. We still do not have support for this on other programming languages as python,c#,etc. So i would expand this part with better explanation